charts/ocis/docs/values.adoc.yaml

```diff
@@ -2231,6 +2231,8 @@ services:
       apps: []
       # -- Configure the {"external_apps": []} section in the Web config.json.
       externalApplications: []
```

Comment:
ideally this should not be needed anymore, instead do anything that needs config via the apps yaml

Reply:
@d7oc and I decided to kill the option to configure apps via `external_apps` directly and instead only use the `apps.yaml` way of providing config for apps.